packages/chntpw/build.sh

```diff
@@ -0,0 +1,20 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/minacle/chntpw
+TERMUX_PKG_DESCRIPTION="chntpw can let you to view and change user passwords, change user/group memberships in SAM file."
```

Comment:
One of the other package descriptions indexed by Repology for this project might be more descriptive.

```suggestion
TERMUX_PKG_DESCRIPTION="Offline Windows NT Password & Registry Editor"
```